perf/lua/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+add_custom_target(test-lua-perf
+                  DEPENDS 1mops_write_perftest
+                          box_select_perftest
+                          uri_escape_unescape_perftest
+                  COMMENT "Running Lua performance tests"
+)
+
+set(TARANTOOL_BIN ${PROJECT_BINARY_DIR}/src/tarantool)
+
+add_custom_target(1mops_write_perftest
+                  COMMAND ${TARANTOOL_BIN} 1mops_write.lua
+                  DEPENDS tarantool
+                  COMMENT Running 1mops_write_perftest
+                  WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/perf/lua
```

Comment:
Nit: this can be a variable, since you use it widely here.

Reply:
Ignored, because I've added a function that generates targets with tests.